services/application.js

```diff
@@ -198,12 +198,7 @@ class Application {
             .verifyInitialized()
             .verifyPermission(USER_PERMISSION_CONSTANTS.SUBMISSION_REQUEST.SUBMIT);
         const application = await this.getApplicationById(params._id);
-        let validStatus = [];
-        if (context?.userInfo?.role === USER.ROLES.SUBMITTER) {
-            validStatus = [NEW, IN_PROGRESS];
-        } else if (context?.userInfo?.role === USER.ROLES.FEDERAL_LEAD) {
-            validStatus = [INQUIRED, IN_PROGRESS];
-        }
+        const validStatus = [IN_PROGRESS, INQUIRED]; //updated based on new requirement.
```

Comment:
Can I see the user-story related to this?

Reply:
Young, there is no user story specific about this, but as we moved to PBAC, this status check should not be based on roles anymore.

Reply:
> Can I see the user-story related to this? I can't find.

Young, there is no user story specific about this, but as we moved to PBAC, this status check should not be based on roles anymore.

Reply:
Thanks.